server/lib/notifications/index.ts

```diff
@@ -5,7 +5,8 @@ export enum Notification {
   MEDIA_PENDING = 2,
   MEDIA_APPROVED = 4,
   MEDIA_AVAILABLE = 8,
-  TEST_NOTIFICATION = 16,
+  MEDIA_FAILED = 16,
+  TEST_NOTIFICATION = 32,
```

Comment:
This is acceptable here because we aren't using these yet to determine which notifications are enabled/disabled... but normally we would never want to change the values here.

Reply:
Gotcha yeah, I hadn't considered that.